packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -136,6 +183,32 @@ export const TimePicker = (props: TimePickerProps): JSX.Element => {
     secondRenderer = (date: Date) =>
       date.getSeconds().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
     meridiemRenderer = (meridiem: MeridiemType) => meridiem,
+    useCombinedRenderer = false,
+    combinedRenderer = (date: Date, precision: Precision) => {
```

Comment:
To me, it would look cleaner if you moved this default `combinedRenderer` value out of this props spread.

Reply:
Yes it would!

Reply:
I moved it out and up in the file as `defaultCombinedRenderer` (line 78)